optimizer_wrapper.rb

```diff
@@ -174,7 +174,7 @@ def self.define_process(service_vrp, job = nil, &block)
     vrp = service_vrp[:vrp]
     dicho_level = service_vrp[:dicho_level].to_i
     split_level = service_vrp[:split_level].to_i
-    log "--> define_process VRP (service: #{vrp.services.size}, shipment: #{vrp.shipments.size}, vehicle: #{vrp.vehicles.size}, v_limit: #{vrp.resolution_vehicle_limit}) with levels (dicho: #{dicho_level}, split: #{split_level})", level: :info
+    log "--> define_process VRP (service: #{vrp.services.size}, vehicle: #{vrp.vehicles.size}, v_limit: #{vrp.resolution_vehicle_limit}) with levels (dicho: #{dicho_level}, split: #{split_level})", level: :info
```

Comment:
Maybe we can keep the count of shipments in these logs, what do you think? It kinda is a valuable info when we need to check the logs (since shipment relations translate into different type of "service").

Reply:
This might output conflicting values I think : vrp.services will include vrp[:shipments]. Vrp.shipments does not exist anymore.

Reply:
Do you find the following confusing? 

`services: 10, shipment relations: 5` which means there are 5 shipments and 0 normal services
`services: 10, shipment relations: 1` which means there is only one P&D pair and remaining 8 services are normal

Now that it is possible to send P&D from mt-web, it is useful to have shipment count in the logs for quick debugging. I used `service: x, shipment: y` logs before and I don't think it can output conflicting values.

Reply:
Oh, right, if we write shipment **relations** then it is clear :) friday afternoon...